src/main/java/org/threeten/extra/chrono/InternationalFixedDate.java

```diff
@@ -0,0 +1,624 @@
+     * @throws DateTimeException if the value of any field is out of range,
+     *                           or if the day-of-month is invalid for the month-year
+     */
+    public static InternationalFixedDate of (final int prolepticYear, final int month, final int dayOfMonth) {
+        return new InternationalFixedDate (prolepticYear, month, dayOfMonth);
+    }
+
+    //-----------------------------------------------------------------------
+
+    /**
+     * Obtains a {@code InternationalFixedDate} from a temporal object.
+     * <p>
+     * This obtains a date in the International fixed calendar system based on the specified temporal.
+     * A {@code TemporalAccessor} represents an arbitrary set of date and time information,
+     * which this factory converts to an instance of {@code InternationalFixedDate}.
+     * <p>
+     * The conversion typically uses the {@link ChronoField#EPOCH_DAY EPOCH_DAY}
+     * field, which is standardized across calendar systems.
+     * <p>
+     * This method matches the signature of the functional interface {@link TemporalQuery}
+     * allowing it to be used as a query via method reference, {@code InternationalFixedDate::from}.
+     *
+     * @param temporal the temporal object to convert, not null
+     * @return the date in the International fixed calendar system, not null
+     * @throws DateTimeException if unable to convert to a {@code InternationalFixedDate}
+     */
+    public static InternationalFixedDate from (final TemporalAccessor temporal) {
+        if (temporal instanceof InternationalFixedDate) {
+            return (InternationalFixedDate) temporal;
+        }
+
+        return InternationalFixedDate.ofEpochDay (temporal.getLong (ChronoField.EPOCH_DAY));
+    }
+
+    /**
+     * Obtains a {@code InternationalFixedDate} representing a date in the International fixed calendar
+     * system from the proleptic-year and day-of-year fields.
+     * <p>
+     * This returns a {@code InternationalFixedDate} with the specified fields.
+     * The day must be valid for the year, otherwise an exception will be thrown.
+     *
+     * @param prolepticYear the International fixed proleptic-year
+     * @param dayOfYear     the International fixed day-of-year, from 1 to 371
+     * @return the date in International fixed calendar system, not null
+     * @throws DateTimeException if the value of any field is out of range,
+     *                           or if the day-of-year is invalid for the year
+     */
+    static InternationalFixedDate ofYearDay (final int prolepticYear, final int dayOfYear) {
+        ChronoField.YEAR.checkValidValue (prolepticYear);
+        InternationalFixedChronology.DAY_OF_YEAR_RANGE.checkValidValue (dayOfYear, ChronoField.DAY_OF_YEAR);
+        boolean leap = InternationalFixedChronology.INSTANCE.isLeapYear (prolepticYear);
+
+        if (dayOfYear > (InternationalFixedChronology.DAYS_IN_YEAR + 1) && !leap) {
```

Comment:
You can use `=` here, as you already check for values greater than.  (If `DAYS_IN_YEAR` actually contained the correct value, this particular check would return the wrong results).